src/index.html

```diff
@@ -16,7 +21,484 @@
       defer
     ></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body id="body">
+    <header
+      class="header"
+      id="home"
+    >
+      <nav class="navigation">
+        <a
+          href="#home"
+          class="navigation__logo-link"
+        >
+          <img
+            src="./images/icons/logo.svg"
+            alt="Logo The Met"
+          />
+        </a>
+        <div class="navigation__contact">
+          <a
+            href="tel:+12125357710"
+            class="navigation__icon navigation__icon--phone"
+          ></a>
```

Comment:
It is good if links that have only icons include a tooltip and aria-label for better accessibility.